connectors/citrus-openapi/src/main/java/org/citrusframework/openapi/actions/OpenApiClientResponseActionBuilder.java

```diff
@@ -90,36 +156,27 @@ public Message build(TestContext context, String messageType) {
             }
 
             if (operation == null) {
-                throw new CitrusRuntimeException("Unable to locate operation with id '%s' in OpenAPI specification %s".formatted(operationId, openApiSpec.getSpecUrl()));
+                throw new CitrusRuntimeException(
+                    "Unable to locate operation with id '%s' in OpenAPI specification %s".formatted(
+                        operationId, openApiSpec.getSpecUrl()));
             }
 
             if (operation.responses != null) {
-                OasResponse response = Optional.ofNullable(operation.responses.getItem(statusCode))
```

Comment:
I moved this implementation into: fillMessageFromResponse so that i can also use it from within the simulator